ted/gtfs.py

```diff
@@ -88,6 +88,69 @@ def fetch_mobility_database() -> pandas.DataFrame:
     return pandas.read_csv(MOBILITY_CATALOG_URL)
 
 
+def check_routes_in_gtfs(gtfs_folder: str):
+    """Get a summary of total stops, total unique stops, and total invalid feeds for each dated entry
+
+    Args:
```

Comment:
Can you follow the template used for docstrings in say `feed_versions_id_and_dates()` (in the same file). Specifically, the "Parameters" and "Returns" format with the right indenting.

Reply:
Changed, it returns nothing so I left out the returns section, should I leave it blank instead?